packages/constants/src/index.ts

```diff
@@ -96,6 +116,7 @@ export type DefinitionType<K extends DefinitionName> = DefinitionTypes[K]
 export type PublishedDefinitions = PublishedRecord<DefinitionName>
 
 export const definitions: PublishedDefinitions = {
+  alsoKnownAs: 'kjzl6cwe1jw148ah60m6n4gwudo8mje9w9p06w3b8bf2g2va3i8me36nazzurc8',
```

Comment:
Should this be `alsoKnownAs` or `aka`?

Reply:
IMO the definition should use the same name as the schema (except with lowercase first character) for consistency and easier discovery, but that's not a strict requirement.

Reply:
ok, fair enough